namespaces/cloud-platform-live-0.k8s.integration.dsd.io/laa-great-ideas-uat/resources/main.tf

```diff
@@ -25,3 +25,37 @@ resource "kubernetes_secret" "ecr-repo" {
     secret_access_key = "${module.ecr-repo.secret_access_key}"
   }
 }
+
+variable "cluster_name" {}
+variable "cluster_state_bucket" {}
+
+module "laa-great-ideas-rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=3.0"
+
+  cluster_name           = "${var.cluster_name}"
+  cluster_state_bucket   = "${var.cluster_state_bucket}"
+  team_name              = "laa-great-ideas"
+  business-unit          = "laa"
+  application            = "laa-great-ideas"
+  is-production          = "false"
+  environment-name       = "uat"
+  infrastructure-support = "laa-great-ideas@digital.justice.gov.uk"
+  db_engine              = "postgres"
+  db_engine_version      = "10.5"
+  db_name                = "laa-great-ideas-db"
```

Comment:
just one nitpick: if defined, db_name must not contain dashes, weird API restriction, so either undefined or laa_great_ideas_db will do